### PR TITLE
Create 'Contact' page

### PR DESCRIPTION
**Pull request by igorstalmach**
- created 'contact page'
- left out 'where to find us' section, because of the recent location change

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -51,6 +52,7 @@ function App() {
                 <ReactCustomScrollbars autoHeight autoHeightMin={'100vh'} autoHide>
                     <Routes>
                         <Route path={'/'} element={<MainPage/>} />
+                        <Route path={'/contact'} element={<ContactPage/>} />
```
Comment (JanZmyslowski) [inline src/App.tsx:55]: You don`t need it here, we belove iterate over `menuItems`, so it enough when it is there

File: src/pages/contact-page/ContactPage.tsx [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+            <div className={styles.wrapper}>
+                <div className={styles.header}>
+                    {__('contactPage.header')}
+                </div>
+                <div className={styles.divider}></div>
+                <div className={styles.title}>
+                    {__('contactPage.title')}
+                </div>
+                <div className={styles.contactWrapper}>
+                    <PersonPicture image={WiktoriaMrowiec} title={__('contactPage.prPresident')} name={'Wiktoria Mrowiec'} mail={'wiktoriamrowiec@gmail.com'}/>
```
Comment (JanZmyslowski) [inline src/pages/contact-page/ContactPage.tsx:25]: I think that we should have list of persons as a list and render them from it

**Review by JanZmyslowski — CHANGES_REQUESTED** [review]

File: src/App.tsx (hunk 1 above)
Comment (igorstalmach) [reply src/App.tsx:55]: done, fixed in other files too

**Comment by JanZmyslowski** [comment]
Conflicts

**Review by JanZmyslowski — APPROVED** [review]

**Pull request merged**